src/outpack/location.py

```diff
@@ -85,3 +239,15 @@ def _location_check_exists(root, name):
 
 def _location_exists(root, name):
     return name in outpack_location_list(root)
+
+
+def _location_driver(location_name, root):
+    location = root.config.location[location_name]
+    if location.type == "path":
+        return OutpackLocationPath(location.args["path"])
+    elif location.type == "http":  # pragma: no cover
```

Comment:
these two should be pretty easy to test for, no? You can access this private function in tests I believe

Reply:
Yeah just didn't really seem much point when it'd just add a test which would immediately be removed in a later PR